doc/source/whats_new.rst

```diff
@@ -18,6 +18,8 @@ Changelog
 
     - Add support for scaling and adjusting the number of channels/time per view by `Jaakko Leppakangas`_
 
+    - Add ``plot_topo`` method to ``Evoked`` object which deprecates ``mne.viz.plot_topo`` by `Jaakko Leppakangas`_
```

Comment:
this is not true. I would write:

Add ``plot_topo`` method to ``Evoked`` object by `Jaakko Leppakangas`_

and in an API section.

Rename and deprecate ``mne.viz.plot_topo`` for ``mne.viz.plot_evoked_topo`` by `Jaakko Leppakangas`_